charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -150,14 +153,14 @@ spec:
         - --cacert=/var/etcd/ssl/ca/ca.crt
         - --insecure-transport=false
         - --insecure-skip-tls-verify=false
-        - --endpoints=https://{{ .Values.name }}-local:{{ .Values.etcd.clientPort }}
+        - --endpoints=https://0.0.0.0:{{ .Values.etcd.clientPort }}
```

Comment:
Please revert this. `{{ .Values.name }}-local` is resolved to `127.0.0.1`.

Reply:
Done